src/openloco/ui/dropdown.cpp

```diff
@@ -90,6 +115,31 @@ namespace openloco::ui::dropdown
         call(0x004CC807, regs);
     }
 
+    // 0x004CC989
+    void show_below(window* window, widget_index widgetIndex, int8_t count, int8_t height)
```

Comment:
`show_below` doesn't make much sense, as it's also used by the bottom toolbars

Reply:
Any suggestions?

Reply:
Deferring to later PR.